BrainPortal/app/models/boutiques_portal_task.rb

```diff
@@ -622,4 +628,24 @@ def file_access_symbol
     @_file_access ||= (self.class.properties[:readonly_input_files].present? || self.tool_config.try(:inputs_readonly) ? :read : :write)
   end
 
+  private
+
+  # Prepare an array with revision information of
+  # all the Boutiques integrator modules used by the
+  # tools.
+  def boutiques_module_information #:nodoc:
+    descriptor = self.descriptor_for_final_task_list
+
+    integrator_modules = (descriptor['custom'] && descriptor['custom']['cbrain:integrator_modules']) || {}
+
+    module_information = []
```

Comment:
This variable is initialized for no reason, it's reset in the map just the line below.

Reply:
And then there is no point in even having the variable, since the map is the last statement of the method.

The method shoudl just end with

```ruby
integrator_modules.map do |module_name, value|
  blha blah
end # block end
end # method end
```